schemas/IDLs/org.opencb.biodata.models/1.3.0-SNAPSHOT/avro/metadata.avdl

```diff
@@ -0,0 +1,89 @@
+@namespace("org.opencb.biodata.models.metadata")
+
+protocol MetadataProtocol {
+
+     enum SampleSetType {
+        CASE_CONTROL,
+        CASE_SET,
+        CONTROL_SET,
+        PAIRED,
+        //PAIRED_TUMOR,
+        //AGGREGATE,
+        TIME_SERIES,
+        FAMILY,
+        TRIO,
+        //COLLECTION,
```

Comment:
can we just remove these three instead of commenting them out ?

Reply:
same as above